### PR TITLE
Created status bar docs

### PR DESCRIPTION
**Pull request by pictos**
The feature is already merged on MCT.

- https://github.com/CommunityToolkit/Maui/pull/617

### CODE REVIEW

File: docs/maui/behaviors/statusbar-behavior.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+### C#
+
+The `StatusbarBehavior` can be used as follows in C#:
+
+```csharp
+class MyPage : ContentPage
+{
+    public MyPage()
+    {
+        this.Behaviors(new IconTintColorBehavior
```
Comment (pictos) [inline docs/maui/behaviors/statusbar-behavior.md:40]:
```suggestion
        this.Behaviors(new StatusbarBehavior
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
+---
+title: StatusbarBehavior - .NET MAUI Community Toolkit
+author: pictos
+description: "The StatusbarBehavior is a behavior allows you to control the statusbar's style."
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:4]:
```suggestion
description: "The StatusBarBehavior is a behavior allows you to control the statusbar's style."
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
+---
+title: StatusbarBehavior - .NET MAUI Community Toolkit
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:2]:
```suggestion
title: StatusBarBehavior - .NET MAUI Community Toolkit
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+author: pictos
+description: "The StatusbarBehavior is a behavior allows you to control the statusbar's style."
+ms.date: 09/27/2022
+---
+
+# StatusbarBehavior
+
+The `StatusbarBehavior` allows you to customize the color and style of yours device statusbar.
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:10]:
```suggestion
The `StatusBarBehavior` allows you to customize the color and style of yours device statusbar.
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
+---
+title: StatusbarBehavior - .NET MAUI Community Toolkit
+author: pictos
+description: "The StatusbarBehavior is a behavior allows you to control the statusbar's style."
+ms.date: 09/27/2022
+---
+
+# StatusbarBehavior
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:8]:
```suggestion
# StatusBarBehavior
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+The `StatusbarBehavior` allows you to customize the color and style of yours device statusbar.
+
+## Syntax
+
+### XAML
+
+The `StatusbarBehavior` can be used as follows in XAML:
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:16]:
```suggestion
The `StatusBarBehavior` can be used as follows in XAML:
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+    <ContentPage.Behaviors>
+        <toolkit:StatusbarBehavior StatusBarColor="Fuchsia" StatusBarStyle="LightContent" />
+    </ContentPage.Behaviors>
+
+</ContentPage>
+```
+
+### C#
+
+The `StatusbarBehavior` can be used as follows in C#:
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:33]:
```suggestion
The `StatusBarBehavior` can be used as follows in C#:
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+-----
+
+## Properties
+
+|Property  |Type  |Description  |
+|---------|---------|---------|
+| StatusBarColor | Color | The `Color` name from the Microsoft.Maui.Graphics namespace. |
+| StatusBarStyle | StatusBarStyle | The style used by statusbar, can be LightContent, DarkContnet or default. |
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:85]:
```suggestion
| StatusBarStyle | StatusBarStyle | The style used by statusbar, can be LightContent, DarkContent or Default. |
```

**Review by jfversluis — CHANGES_REQUESTED** [review]
So many B's........

File: docs/maui/TOC.yml [hunk 9]
```diff
@@ -47,6 +47,8 @@ items:
       href: behaviors/required-string-validation-behavior.md
     - name: "SetFocusOnEntryCompletedBehavior"
       href: behaviors/set-focus-when-entry-completed-behavior.md
+    - name: "StatusbarBehavior"
```
Comment (jfversluis) [inline docs/maui/TOC.yml:50]:
```suggestion
    - name: "StatusBarBehavior"
```

File: docs/maui/behaviors/statusbar-behavior.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+| StatusBarColor | Color | The `Color` name from the Microsoft.Maui.Graphics namespace. |
+| StatusBarStyle | StatusBarStyle | The style used by statusbar, can be LightContent, DarkContent or Default. |
+
+## Examples
+
+You can find an example of this behavior in action in the [.NET MAUI Community Toolkit Sample Application](https://github.com/CommunityToolkit/Maui/blob/main/samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/StatusBarBehaviorPage.xaml).
+
+## API
+
+You can find the source code for `StatusbarBehavior` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/StatusBar/StatusBarBehavior.shared.cs).
```
Comment (jfversluis) [inline docs/maui/behaviors/statusbar-behavior.md:93]:
```suggestion
You can find the source code for `StatusBarBehavior` over on the [.NET MAUI Community Toolkit GitHub repository](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/StatusBar/StatusBarBehavior.shared.cs).
```

**Comment by jfversluis** [comment]
OK now I'm done, I swear

**Review by jfversluis — APPROVED** [review]
Awesome job @pictos, thanks so much!

**Pull request merged**